client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -102,7 +102,7 @@
         <dp-data-table
           ref="dataTable"
           class="overflow-x-auto"
-          :class="isFullscreen ? 'px-2 overflow-y-scroll h-5/6 grow' : '-mt-3'"
+          :class="isFullscreen ? 'px-2 overflow-y-scroll grow' : '-mt-3'"
```

Comment:
i did not mean to apply it here. when you merge release into this branch, you'll see a new dom element with `ref="scrollBar"`. this element has a negative margin applied, which needs to be made conditional.

Reply:
what do you mean by "merge release into this branch"? We didn't have a FF merge. I merged the main branch with your changes, and this margin `-mt-3` helped to remove an extra toolbar below

Reply:
ok, but if you think it better to apply it there, I change it ;)

Reply:
no need to!